src/cleed++/IVCurvePair.cc

```
@@ -0,0 +1,123 @@
+/*!
+ * \file
+ * \author Liam Deacon
+ *
+ * Implements IVCurvePair class.
+ */
+
+#include <IVCurvePair.hh>
+
+using namespace cleed;
+
+IVCurvePair::IVCurvePair(const IVCurvePair &other) {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: iv_pair

(cppcoreguidelines-pro-type-member-init)